packages/xrpc-server/tests/bodies.test.ts

```diff
@@ -237,4 +245,18 @@ describe('Bodies', () => {
 
     await expect(promise).rejects.toThrow('request entity too large')
   })
+
+  it('requires any parsable Content-Type for blob uploads', async () => {
+    // not a real mimetype, but correct syntax
+    await client.call('io.example.blobTest', {}, randomBytes(BLOB_LIMIT), {
+      encoding: 'some/thing',
+    })
+
+    // empty mimetype, but correct syntax
+    const p3 = client.call('io.example.blobTest', {}, randomBytes(BLOB_LIMIT), {
+      encoding: '',
+    })
+    // TODO: await expect(p3).rejects.toThrow('Request encoding (Content-Type) required but not provided')
```

Comment:
i know what the issue is here btw, i'll push a fix

Reply:
alright well the issue is that xrpc/client is defaulting that value to 'application/json' which I think is good in the general case (I can't think of a situation where we should like clients send an empty content type & nearly all of our methods are json 🤔)

So i tried deconstructing it to basic fetch semantics, but it's only working for me intermittently. Oddly it works fine when the test before it is commented out.

```ts
    // empty mimetype, but correct syntax
    const res = await fetch(`${url}/xrpc/io.example.blobTest`, {
      method: 'post',
      headers: { 'Content-Type': '' },
      body: randomBytes(BLOB_LIMIT),
      // @ts-ignore see note in @atproto/xrpc/client.ts
      duplex: 'half',
    })
    const resBody = await res.json()
    const status = res.status
    expect(status).toBe(400)
    expect(resBody.error).toBe('InvalidRequest')
    expect(resBody.message).toBe(
      'Request encoding (Content-Type) required but not provided',
    )
```

Reply:
@devinivy any ideas before I spend more time spinning on this?

Reply:
Nothing is coming to mind, sorry!  Happy to pair on it if you want another set of eyes, though.

Reply:
JSON default makes sense!

I was also running in to some weird "what the test before is doing" behavior and that was when I gave up. Don't think we should collectively burn too many cycles on this right now, we can just mark the test with a skip or something.

Reply:
yeah I'd say copy this into a new test & skip it for now with a comment about why it's not working